sunxspex/acgaunt.py

```diff
@@ -0,0 +1,361 @@
+import numpy as np 
+
+def acgaunt(wave, te_6):
```

Comment:
Perhaps this should be a class where the data currently hard coded in the function is in a (or a few) files.  On instantiation of the class the files can be read once and then the function can be a class method which has access to the data without having to reread it every time.

Reply:
Yep _definitely_ agree, at the moment it's literally just a direct copy of IDL --> Python 😋

Reply:
Great!  We're on the same page!